crmsh/bootstrap.py

```diff
@@ -235,6 +235,24 @@ def wait_for_cluster():
     status_done()
 
 
+def pick_default_value(vlist, ilist):
```

Comment:
Hm, this function is a bit strange to me.. only the first element of `ilist` is ever used, and `vlist` is modified.
I would prefer it if the function returned two return values: The chosen default value, and the remaining values from vlist.. and it would be clearer if the `ilist` parameter was a single element called `preferred_value` or something like that..

Also I try to document all functions with a documentation string, especially functions that modify their parameters :)